Update tone of CS docs to be more actionable and streamlined (#16942)
PostHog/posthog.com: contents/handbook/cs-and-onboarding/foundation-check.md

showTitle: true
---

> A checklist to run through before a customer discovery call. PostHog only shows you some of this — for backend implementation details, you'll need to ask on the call.

Reviewer: I think it's worth calling out some of the non-technical discovery worth doing.

```suggestion
Prior to a customer discovery call, review session replays of the user to see how they interact with PostHog, and where improvements can be made. Check historic conversations in Vitally, or calls in BuildBetter, for wider context.

For fundamentals, use this checklist to check for common issues before a customer discovery call. PostHog only shows you some of this — for backend implementation details, you'll need to ask on the call.
```

## Events and event properties

Check what events the customer is tracking, whether they have custom events, whether autocapture is on, and whether they're collecting event properties (custom or autocapture).

Two places to look:

- **Activity** — recent events.
- **Data management** — custom actions, event definitions, property definitions.

> Look for custom actions. Customers without any actions often have their setup wrong. Common patterns: renaming events into something more meaningful (e.g. `purchase_completed` instead of `clicked_purchase_button`), or bundling events like `user signups` or `purchases`.

Reviewer: ```suggestion
Look for custom actions. Customers without any actions often aren't using PostHog effectively. Common patterns: renaming events into something more meaningful (e.g. `purchase_completed` instead of `clicked_purchase_button`), or bundling events like `user signups` or `purchases`.
```

## Ecommerce events

For ecommerce customers, check whether they've implemented the [ecommerce events specification](/docs/data/event-spec/ecommerce-events) — events like `sku`, `product_id`, `category`. Many customers don't know it exists.

## SDK or library version

Check that the customer is on an up-to-date SDK:

- In **Activity**, click `Configure columns` and add `Library` and `Library Version`.
- Or look up the `Library version audit` table in Metabase.

Compare against the latest versions in our GitHub repos.

## Sign up for an account

If the customer's product offers a free account, sign up and walk through the flow. You'll see which events fire, which don't, and what's likely missing.

## Dashboards

Have they set up custom dashboards or insights for their own goals (sign-ups, retention, free-to-paid), or are they relying on defaults? On the discovery call, you can ask whether what they're tracking matches their priorities.

## Data pipelines for event notifications

Check whether they're using data pipeline destinations (e.g. Slack notifications on specific events). Many customers don't realize this use case exists — it's an easy upsell.

## Query failure rate

> Visible in Vitally. A high failure rate means a customer is trying to do something that's not working — bring it up on the discovery call.

Reviewer: I'm partial to removing this, I've found it doesn't correlate well to lived customer experience.

I'm also not sure how a CSM can really leverage this stat. "We saw your failure rate is 12%" feels a little strange.

You have much more experience with this stat, so I'll default to you if you think we should keep it.

Author: i agree, it's _sometimes_ useful but it's often at the bottom of the list so let's remove. it's also not very actionable.
